reactive-modules/ratpack-rxjava/src/main/java/com/baeldung/rxjava/service/impl/MovieServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.rxjava.service.impl;
```

Comment:
If possible, let's place all of the classes for the same tutorial in the same shared package. Having subpackages is fine, but could we do something like `com.baeldung.ratpack.intro.impl` or `com.baeldung.ratpack.intro.rxjava.service` if we want to be clear that this is rxjava-specific. The important piece is that everything from our tutorial be namespaced to our tutorial so we don't clobber future authors using the same module.